cypress/support/commands.ts

```diff
@@ -106,7 +106,7 @@ export const loginViaDex = (username: string, password: string) => {
               body: {req: req, approval: 'approve'},
             }).then(() => {
               cy.visit('/')
-              cy.getCookie('session').should('exist')
+              // cy.getCookie('session').should('exist')
```

Comment:
why are we commenting this out?

Reply:
Ish, but not really. It's related to this thread:

https://influxdata.slack.com/archives/C01E1H8D04Q/p1625068024309800

Reply:
ah, got it that makes sense.